services/apps/alcs/src/portal/inbox/notice-of-intent/inbox-notice-of-intent.service.ts

```diff
@@ -155,8 +155,8 @@ export class InboxNoticeOfIntentService {
     }
 
     if (searchDto.civicAddress) {
-      query = query.andWhere('parcel.civic_address like :civic_address', {
-        civic_address: `%${searchDto.civicAddress}%`,
+      query = query.andWhere('LOWER(parcel.civic_address) like LOWER(:civic_address)', {
+        civic_address: `%${searchDto.civicAddress}%`.toLowerCase(),
```

Comment:
question: If you are lower casing it in the SQL with LOWER, why do you also lowercase it first?

Reply:
just  consistency with all fields

Reply:
Sorry, I'm not sure what you mean, consistency with which fields?

Reply:
I noticed you resolved this without answer my question, did you miss it?

Reply:
@dhaselhan i am just keeping it consistent with other searches, can remove it from everywhere